### ISSUE TITLE
Wrong syntax creating unique in Oracle 11g

### ISSUE THREAD

**Issue by babbomaria** [serge-rider/dbeaver#61]
When I create a unique with gui

![2015-11-17_075454](https://cloud.githubusercontent.com/assets/344999/11204746/2bd28cea-8d02-11e5-9204-bf12ed8bd18b.jpg)

While the syntax for Primary is ok for unique under oracle is wrong

![2015-11-17_080436](https://cloud.githubusercontent.com/assets/344999/11204758/460c3fe8-8d02-11e5-9573-d03a2df565ab.jpg)

```sql
    ALTER TABLE CBIUSER.SHARED_OBJ ADD CONSTRAINT SHARED_OBJ_UK3 UNIQUE KEY (ID1,ID2,ID3);
```
Should be

```sql
    ALTER TABLE CBIUSER.SHARED_OBJ ADD CONSTRAINT SHARED_OBJ_UK3 UNIQUE (ID1,ID2,ID3);
```

**Comment by serge-rider** [comment]
Thanks for report

**Comment by jencii** [comment]
Same on DB2 (10.5.7)

**Comment by serge-rider** [comment]
Thanks for report.
Fix will be added in 3.6.7
